packages/contracts-bedrock/src/dispute/FaultDisputeGameN.sol

```diff
@@ -960,14 +983,13 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
         // Walk up the DAG until the ancestor's depth is equal to the split depth.
         uint256 currentDepth;
```

Comment:
This function should also be updated. Especially for line 1002, `wasAttack` should be replaced by `attachBranch` and then use the parameter to find the starting and disputed outputs accordingly.